src/lintlist.rs

```diff
@@ -0,0 +1,2274 @@
+/// Lint data parsed from the Clippy source code.
+#[derive(Clone, PartialEq, Debug)]
+pub struct Lint {
+    pub name: &'static str,
+    pub group: &'static str,
+    pub desc: &'static str,
+    pub deprecation: Option<&'static str>,
+    pub module: &'static str,
+}
+
+pub const ALL_LINTS: [Lint; 320] = [
+    Lint {
+        name: "too_many_arguments",
```

Comment:
Can we sort these lints by name? I think that would look better in the help message.